pyproject.toml

```diff
@@ -105,17 +105,19 @@ structlog = ["structlog"]
 [project.scripts]
 litestar = "litestar.__main__:run_cli"
 
-
 [tool.hatch.build.targets.sdist]
 include = [
   'docs/PYPI_README.md',
   '/litestar',
 ]
 
-
 [tool.uv]
 default-groups = ["dev", "linting", "test", "docs"]
 
+#[tool.uv.sources]
+# Only needed when working on the theme; also need to uncomment docs package
+#litestar-sphinx-theme = { path = "../litestar-sphinx-theme", editable = true }
+
```

Comment:
it feels weird to have to comment / uncomment to work on docs, or there's something I dont get, isn't there a better way ?

Reply:
Yes, I think you can use something like [this](https://github.com/litestar-org/litestar-htmx/blob/1be9b70db5208aedc9e46c521f55f2416ed2dd29/pyproject.toml#L64-L65)

Reply:
Would this work? The issue I am fighting and that this solves currently is that I can have a locally checked out copy of `litestar-sphinx-theme` and can uncomment this section and the one in the `docs = [...]` group so that changes can be seen live across the project without having to push upstream or something.

Reply:
Yep.  I think this should handle this case.